### ISSUE TITLE
Add support for preprocessed CSS languages

### ISSUE THREAD

**Issue by ebramanti** [xekri/vscode-styled-jsx#7]
`styled-jsx` supports [plugins](https://github.com/zeit/styled-jsx#example-plugins) which allows for the use of multiple different languages (Stylus, Sass, PostCSS, etc.)

It would be cool to support multiple languages out of the box and be able to specify the language used in a project-specific configuration.

**Comment by johnlombardo-dev** [comment]
Agreed! `styled-jsx` plugins like [styled-jsx-plugin-sass](https://github.com/giuseppeg/styled-jsx-plugin-sass) work well, but not if you want syntax highlighting. For now I have stopped using SCSS syntax because `vscode-styled-jsx` is more important 😄

**Comment by samuelroy** [comment]
While it would be awesome to have a project-specific configuration and use only this extension, I forked the project to support Stylus syntax highlighting. 

See: https://github.com/samuelroy/vscode-styled-jsx-stylus

**Comment by Soundvessel** [comment]
We use styled-jsx-plugin-sass with Next.js to import and reuse SCSS functions and SCSS variables from customized Bootstrap design systems. The system is used globally but this way we can create scoped CSS in line with the design system. We tried [vscode-styled-jsx-languageserver](https://github.com/Grimones/vscode-styled-jsx-languageserver) but it can not handle how we break out styles as const variables to keep the component function clean. [Issue thread with example code.](https://github.com/Grimones/vscode-styled-jsx-languageserver/issues/12)
